src/WebJobs.Script/Description/PowerShell/PowerShellFunctionInvoker.cs

```diff
@@ -96,8 +95,10 @@ public class PowerShellFunctionInvoker : ScriptFunctionInvokerBase
                         LogLoadedModules();
                     }
 
-                    _script = GetScript(_scriptFilePath);
-                    powerShellInstance.AddScript(_script, true);
+                    if (File.Exists(_scriptFilePath))
+                    {
+                        powerShellInstance.AddScript(_scriptFilePath, false);
```

Comment:
Technically I suppose it could, if the code being invoked was dependent on invocation in a separate scope (i.e. via the call operator instead of via the dot source operator). Are multiple functions being invoked in the same session/runspace? If so, they shouldn't be. I don't think they are though, and it would be a stretch for anything to depend on how it is invoked. Besides, this is experimental so making this change now is the right time to do so, especially when this is a better way to invoke it.